### ISSUE TITLE
How can I add resolve-readmodel-mysql to hacker-news example?

### ISSUE THREAD

**Issue by vitaliibilous** [reimagined/resolve#638]
Hello, I have a problem with implementing mysql adapter to the project.

I tried: 
```
// hacker-news/common/read-models/index.js
import projection from './projection'
import resolvers from './resolvers'

import adapter from 'resolve-readmodel-mysql'

const storiesReadModel = {
  name: 'default',
  projection,
  resolvers,
  adapter
}

export default [storiesReadModel]
```
But the problem occured was 
```
ERROR in ./node_modules/cardinal/lib/highlightFileSync.js
Module not found: Error: Can't resolve 'fs' in ...
```

Same with `tls`, `net` modules of mysql2.

Resolve tries to build with client configuration and not server one.

Can you provide an example how to implement this module correctly?
Many thanks

**Comment by IhostVlad** [comment]
Hello, Vitalii Bilous! Thanks for response
Right now package `resolve-scripts`, which is responsible for client & server bundling, is migrating to new-style declarative config, which will allow to select adapters and specify their options in `resolve.config.json` file.
Due migration process, bug with client bundling for server-only packages is appeared. You can use following workaround. Take in mind that `resolve 0.11.0` will use declarative configs - so bug will desappear.

Temporary workaround for current version:
```javascript
import projection from './projection'
import resolvers from './resolvers'

const storiesReadModel = {
  name: 'default',
  projection,
  resolvers
}
eval (`
  try {
   storiesReadModel.adapter = require ( 'resolve-readmodel-mysql' ) ({
       host: 'localhost',
       user: 'sqluser',
       password: 'sqlpass',
       .....
    })
  } catch (err) { }
`)

export default [storiesReadModel]
```

**Comment by TatyanaRyzh** [comment]
Fixed in [V0.12.1](https://github.com/reimagined/resolve/releases/tag/V0.12.1)
